src/main/java/nextstep/subway/exception/LineNotFoundException.java

```diff
@@ -1,7 +1,9 @@
 package nextstep.subway.exception;
 
+import org.springframework.http.HttpStatus;
+
 public class LineNotFoundException extends SubwayException {
     public LineNotFoundException(Long id) {
-        super("지하철 노선 ID " + id + " 가 존재하지 않습니다.");
+        super("지하철 노선 ID " + id + " 가 존재하지 않습니다.", HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
클라이언트가 잘못된 지하철 노선 id를 전달한 경우는 응답 코드를 400으로 보내는 게 적절하지  않을까요?
```suggestion
        super("지하철 노선 ID " + id + " 가 존재하지 않습니다.", HttpStatus.BAD_REQUEST);
```